rust/agama-lib/src/questions.rs

```diff
@@ -41,48 +41,20 @@ impl GenericQuestion {
     }
 }
 
-/// Specialized question for Luks partition activation
+/// Composition for questions which include password.
+/// TODO: research a bit how ideally do mixins in rust
```

Comment:
This TODO has been resolved by this code already?
Or add pointers to ideas and examples to comments/commit message.

Reply:
I will write it before comment in this PR. I will try to document the idea